tutorials/environmental_data_and_visualization.md

```diff
@@ -0,0 +1,173 @@
+\page environmental_data Visualizing Environment Data
+
+Lot of times you may come accross environment properties that vary with time and
+space. For instance if you'd like to model properties such as gas distribution
+and water temperature one needs to have some way of defining values such that
+the values vary through out the environment. Gazebo's Environment Data
+infrastructure enables such visualization and sensing capabilities.
```

Comment:
@arjo129 meta: it may be worth mentioning that these quantities do not affect physics in any way.

Reply:
I'm holding off on that because one of the items on our M4 is to have the currents effect physics.